master/getting-started/kubernetes/upgrade/downgrade.md

```diff
@@ -3,20 +3,49 @@ title: Downgrading Calico
 no_canonical: true
 ---
 
+## About downgrading {{site.prodname}}
+
 Under some circumstances, you may need to perform a downgrade and return your
-cluster to the previous version of {{site.prodname}}. You may need to do this
-before running `calico-upgrade complete` or afterwards. If you need to downgrade 
-your cluster after running `calico-upgrade complete`, you should do so as soon
-as possible to avoid an outage. Any pods created after `calico-upgrade complete`
-and before downgrading will lose networking.
+cluster to the previous version of {{site.prodname}}. If you need to downgrade
+you should do so as soon as possible to avoid an outage.
+
+> **Important**: After downgrading or aborting the migration it is necessary
+> to delete the previously migrated [etcd](delete)
+> or [Kubernetes API](delete)
+> data before re-running the migration.
+{: .alert .alert-info}
+
+The downgrade procedure varies according to how you originally installed
+{{site.prodname}} and your datastore type.
+
+- Follow the steps to [Downgrade a self-hosted installation that uses the etcd
+  datastore](#downgrading-a-self-hosted-installation-that-uses-the-etcd-datastore)
+  when migration has been done and the {{site.prodname}} components have been
+  upgraded.
+  > **Important**: Any pods created after `calico-upgrade complete` and
+  > before downgrading will lose networking.
+  {: .alert .alert-danger}
+
+- Follow the steps to [Abort the upgrade when using the etcd
```

Comment:
The abort procedure is discussed in the migrate section so I don't think it belongs here, I think we can delete this.

Reply:
Is what you are talking about in a PR still? In master I don't see an abort procedure, the only abort mention is a reference to it in the Errors note.

Reply:
After talking to @tmjd he explained to me more about the `abort` command so it is fine then.